platform/server/plugins/PluginsService.ts

```diff
@@ -54,38 +55,48 @@ export class PluginsService implements CoreService {
    * of plugins.
    */
   private readPlugins() {
-    return readDirAsObservable(this.env.pluginsDir)
-      .mergeMap(dirs => dirs)
-      .map(name => {
-        const pluginPath = this.env.getPluginDir(name);
-        const json = require(pluginPath);
-
-        if (!('plugin' in json)) {
-          throw new Error(
-            `'plugin' definition missing in plugin [${pluginPath}]`
-          );
-        }
+    const { pluginsDir } = this.env;
+
+    return (
+      readDirAsObservable(pluginsDir)
```

Comment:
__question:__ There is no need to change anything in this PR, just wondering if we can we replace `AsObservable` with just `$` as we do for variables?

Reply:
✅